client/blocks/calendar-popover/README.md

```diff
@@ -0,0 +1,85 @@
+Calendar Popover
+================
+
+This component shows a Popover with a calendar inside.
+
+Beyond to combining the Popover and PostSchedule components in only one, this component connects with the state-tree, so in this way timezone data related to the site configuration is propagated automatically.
```

Comment:
maybe:

"Beyond combining [`Popover`](https://github.com/Automattic/wp-calypso/tree/master/client/components/popover)  and [`PostSchedule`](https://github.com/Automattic/wp-calypso/tree/master/client/components/post-schedule), this component connects... "